docs/src/assets/icons/Colorway=Default, State=Default.svg:Zone.Identifier

```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
+ZoneId=3
+ReferrerUrl=C:\Users\hamir\Downloads\Doc Icons.zip

```

Comment:
What is the deal with these files that refer to Hamir's machine?

Reply:
I wondered about that, as well. They're gone now.